tests/network/sriov.go

```diff
@@ -795,3 +796,16 @@ func mountGuestDevice(vmi *v1.VirtualMachineInstance, devName string) error {
 		&expect.BExp{R: console.RetValue("0")},
 	}, 15)
 }
+
+func checkCloudInitMetaData(vmi *v1.VirtualMachineInstance, testFile, testData string) {
+	cmdCheck := "cat " + filepath.Join("/mnt", testFile) + "\n"
+	res, err := console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
+		&expect.BSnd{S: "sudo su -\n"},
+		&expect.BExp{R: console.PromptExpression},
+		&expect.BSnd{S: cmdCheck},
+		&expect.BExp{R: testData},
+	}, 15)
+	if err != nil {
+		Expect(res[1].Output).To(ContainSubstring(testData))
+	}
```

Comment:
I understand that this function was copied from `tests/utils.go`, but I do not follow what is there a need to have another assertion in case there was an error.
Could you please elaborate?